usage.md

```diff
@@ -66,6 +67,86 @@ Returns the Smart Address as per ERC-4337, the SCW address, associated with the
 
 Returns the balance available in the gas tank that can be utilized via the current user's SCW address.
 
+### `createSession()`
+
+Creates session keys using the [`CreateSessionParam`](https://gasless-sdk-ref-guide.netlify.app/types/createsessionparam) policy and requests the user for permission. When the user initiates any app operation that fulfills the session policy, it will be autonomously approved.
```

Comment:
Should we mention SCW method name i.e. `createSession` instead of its parameter type ?

Reply:
Yes indeed my bad.

Reply:
fixed